mlrun/common/schemas/model_monitoring/constants.py

```diff
@@ -228,6 +229,12 @@ class MonitoringFunctionNames(MonitoringStrEnum):
     WRITER = "model-monitoring-writer"
 
 
+class V3IOTSDBTables(MonitoringStrEnum):
```

Comment:
It spouse to be valid for all the tsdb's engines?  
```suggestion
class MonitoringTSDBTables(MonitoringStrEnum):
```